src/task-status.js

```diff
@@ -0,0 +1,8 @@
+const tasks = JSON.parse(localStorage.getItem('tasks'));
+
+function updateStatus(index, completed) {
+  tasks[index].completed = completed;
+  localStorage.setItem('tasks', JSON.stringify(tasks));
+}
```

Comment:
- [ ] when trying to change the task status the user receives the below error, please fix this before requesting another review.
![afbeelding](https://user-images.githubusercontent.com/39240820/236194432-d7e7fd43-22a8-4dba-a387-0b90830066e8.png)